File: S3/AA/AA_TP1_Python/test_truncate.py
import unittest
import strlen


class MyTestCase(unittest.TestCase):

    def test_truncate(self):
        li = ["a", "bb", "ccc", "dddd", "eeeee"]
        strlen.truncate(li, 3)
        self.assertEquals(["a","bb","ccc","ddd","eee"], li)


if __name__ == '__main__':
    unittest.main()

File: S3/AA/AA_TP1_Java_Morgado-Samagaio_Tandol/Workspace/TP1/src/tp1/tests/TestTruncate.java
package tp1.tests;

import static org.junit.Assert.assertEquals;

import java.util.Arrays;
import java.util.List;

import org.junit.Test;

import tp1.StrLen;

public class TestTruncate {

	@Test
	public void Test() {
		List<String> input = Arrays.asList("a", "bb","ccc","dddd","eeeee");
		StrLen.truncate(input,3);
		List<String> expected = Arrays.asList("a","bb","ccc","ddd","eee");
		assertEquals(expected, input);
	}
	
}
